src/channel.cpp

```diff
@@ -71,6 +71,49 @@ Channel::~Channel() {
     assert(participants.size() == 0);
 }
 
+// Minor utils defined here until we have a utility function file.
+
+/**
+* Change a string's pointer value to a lowercased version.
+* @param string& str string pointer reference
+* @returns void
+*/
+void setLower(string& str) {
+  int intLength;
+  const size_t length = str.length();
+
+  if (length > INT_MAX) {
+    LOG(WARNING) << "String size_t const exceeds maximum signed integer size. Overflow immenent."; // For the rare, rare case that this may occur.
+  }
+
+  intLength = static_cast<int>(length);

```

Comment:
Size_t conversions to int produce compiler warnings.

"warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data"